identity-service/src/notifications/notificationQueue.js

```diff
@@ -251,13 +255,25 @@ async function drainPublishedMessages(logger, optimizelyClient) {
   return numProcessedNotifs
 }
 
-async function drainPublishedSolanaMessages(logger) {
+async function drainPublishedSolanaMessages(logger, optimizelyClient) {
   logger.info(
     `[notificationQueue:drainPublishedSolanaMessages] Beginning processing of ${pushNotificationQueue.PUSH_SOLANA_NOTIFICATIONS_BUFFER.length} notifications...`
   )
 
   let numProcessedNotifs = 0
   for (const bufferObj of pushNotificationQueue.PUSH_SOLANA_NOTIFICATIONS_BUFFER) {
+    const notificationMappingVar =
+      notificaitonTypeMapping[bufferObj.notification.type]
+    const isDisabled = getRemoteFeatureVarEnabled(
+      optimizelyClient,
+      DISCOVERY_NOTIFICATION_MAPPING,
+      notificationMappingVar
+    )
+
+    if (isDisabled === true) {
+      return
+    }
```

Comment:
can you go over why we have to check `isDisabled` for a notification here and in processNotifiation()? are they two separate flows?

Reply:
the two places we check in identity are for the two notifications queues - solana and poa notifications. 
Otherwise the checks for the remote config val are from the plugin. 
We want to be able to flip a switch so identity stops push notifications and the plugin starts